config/splits/vpge/core.entity_form_display.config_pages.stanford_basic_site_settings.default.yml

```diff
@@ -17,7 +17,6 @@ dependencies:
     - field.field.config_pages.stanford_basic_site_settings.su_site_algolia_index
     - field.field.config_pages.stanford_basic_site_settings.su_site_algolia_key
     - field.field.config_pages.stanford_basic_site_settings.su_site_algolia_search
-    - field.field.config_pages.stanford_basic_site_settings.su_site_algolia_terms
```

Comment:
Why remove this? It's part of the downstream changes: https://github.com/SU-SWS/stanford_profile/blob/60aa0df07293b5f92aa837e8ab2c335be46a8f0c/config/sync/core.entity_form_display.config_pages.stanford_basic_site_settings.default.yml#L20